opentelemetry-user-events-logs/src/logs/exporter.rs

```diff
@@ -55,6 +98,14 @@ impl UserEventsExporter {
         })
     }
 
+    /// Build a log processor for the `UserEventsExporter` using the provided options.
+    pub fn build_processor(
+        options: UserEventsOptions,
+    ) -> ReentrantLogProcessor<UserEventsExporter> {
+        let exporter = UserEventsExporter::new(options).expect("Failed to create exporter");
```

Comment:
now that we have options and options_builder, lets handle failure within that only, so the new(options) does not have to return Result?